storage/bucket_test.go

```diff
@@ -142,11 +142,11 @@ func TestBucketAttrsToRawBucket(t *testing.T) {
 					Type: DeleteAction,
 				},
 				Condition: &raw.BucketLifecycleRuleCondition{
-					Age:                 30,
-					IsLive:              googleapi.Bool(true),
-					CreatedBefore:       "2017-01-02",
-					MatchesStorageClass: []string{"NEARLINE"},
-					NumNewerVersions:    10,
+					DaysSinceNoncurrentTime: 30,
+					IsLive:                  googleapi.Bool(true),
+					NoncurrentTimeBefore:    "2017-01-02",
+					MatchesStorageClass:     []string{"NEARLINE"},
+					NumNewerVersions:        10,
```

Comment:
Is there a system test you could update?

Reply:
Yep, updated the system test for Lifecycle to exercise the new options and added a simple one for CustomTime